packages/template-retail-react-app/scripts/compile-translations.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* eslint @typescript-eslint/no-var-requires: "off" */
+const {exec} = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+const packagePath = path.join(process.cwd(), 'package.json')
+const pkgJSON = JSON.parse(fs.readFileSync(packagePath))
+
+const overridesDir = pkgJSON.ccExtensibility?.overridesDir
+const outputFolder = overridesDir
+    ? path.join(overridesDir, 'app/static/translations/compiled')
+    : 'app/static/translations/compiled'
+
+const inputFolder = process.argv[2]
+
+try {
+    const command = `formatjs compile-folder --ast ${inputFolder} ${outputFolder}`
+    console.log('Compiling translations into the folder:', outputFolder)
```

Comment:
These two scripts are quite similar, is it easier to combine both into one single script? E.g 
`"compile-translations:pseudo": "node ./scripts/compile-pseudo-translation.js translations/en-US.json --psedo-locale=en-XA"`
```
const psedoLocale = process.argv[2]
let command
if (!psedo-locale) {
   command = `formatjs compile-folder --ast ${inputFolder} ${outputFolder}`

} else {
    const locale = psedo-locale
    const outputFile = `${outputFolder}/${locale}.json`
    command = `formatjs compile --ast ${inputFile} --out-file ${outputFile} --pseudo-locale ${locale}`
}
exec(command, ...)
```

Reply:
I've done some refactoring to extract out the duplicate code into its own file.